src/python/issue_management/issue_tracker_utils.py

```diff
@@ -170,20 +97,31 @@ def get_similar_issues(testcase,
         continue
 
       # Get issue object using ID.
-      issue = issue_tracker_manager.get_issue(issue_id)
+      issue = issue_tracker.get_issue(issue_id)
       if not issue:
         continue
 
       # If our search criteria allows open bugs only, then check issue and
       # testcase status so as to exclude closed ones.
-      if (can == IssueTrackerManager.CAN_OPEN and
-          (not issue.open or not testcase.open)):
+      if (only_open and (not issue.is_open or not testcase.open)):
         continue
 
-      issue_objects.append(issue)
+      issues.append(issue)
       issue_ids.append(issue_id)
 
-  return issue_objects
+  return issues
+
+
+def get_similar_issues_url(issue_tracker, testcase, only_open=True):
+  """Get similar issues web URL."""
+  keywords = get_search_keywords(testcase)
+  return issue_tracker.find_issues_url(keywords=keywords, only_open=only_open)
+
+
+def get_issue_url(testcase):
+  """Return issue url for a testcase."""
+  issue_tracker = get_issue_tracker_for_testcase(testcase)
+  return issue_tracker.issue_url(testcase.bug_information)
```

Comment:
Should we check if testcase.bug_information exists, otherwise return None or empty string ?

Reply:
Done!